### DISCUSSION TITLE
After configuring useAI in the czrc file, other options are no longer available.

### DISCUSSION THREAD

**Discussion by guotingchao** [Zhengqbbb/cz-git#130] [Q&A]
After entering `cz` on the command line, the AI is only responsible for generating the subject, but none of the other messages are configured to take effect.

```json
 "messages": {
    "type": "选择你要提交的类型 :",
    "scope": "选择一个提交范围（可选）:",
    "customScope": "请输入自定义的提交范围 :",
    "subject": "填写简短精炼的变更描述 :\n",  // Only works here, no other options available 
    "body": "填写更加详细的变更描述（可选）。使用 \"|\" 换行 :\n",
    "breaking": "列举非兼容性重大的变更（可选）。使用 \"|\" 换行 :\n",
    "footerPrefixesSelect": "选择关联issue前缀（可选）:",
    "customFooterPrefix": "输入自定义issue前缀 :",
    "footer": "列举关联issue (可选) 例如: #31, #I3244 :\n",
    "confirmCommit": "是否提交或修改commit ?"
  },
```

**Comment by Zhengqbbb** [comment]
https://twitter.com/zhengqbbb/status/1634662649026969601
or
https://github.com/Zhengqbbb/cz-git/releases/tag/v1.6.0

Hope the video can help u. If have any question please tell me ~ 👀

**Reply by guotingchao** [reply]
Hi there, Thank u for ur reply.
Regarding the previous question, perhaps I did not describe it clearly enough.
After I checked out the video and tried it the same way. it doesn't work.

I just used the global installation of `cz` and `commitizen`, the question should be: can it be done without using `czg`

TIA! 
----

<img width="759" alt="image" src="https://github.com/Zhengqbbb/cz-git/assets/2443075/a2102375-893d-4b7a-90e1-0210a9f4c004">

**Reply by Zhengqbbb** [reply]
Look like you are using `commitizen` + `cz-git`, you cannot use subcmd and option usage like'git cz ai ', because commitizen as a launcher does not provide this way to plugins

How to activate AI mode:
1. Environment variables injection `czai = 1 git cz` or `czai = 1 cz`
2. Change the configuration `useAI: true`

---
If you use `czg` <sup> https://cz-git.qbb.sh/cli/ https://cz-git.qbb.sh/cli/why </sup>. you can use subcmd `git czg ai` or `czg ai`

https://cz-git.qbb.sh/config/engineer#useai
